avm/res/api-management/service/main.bicep

```
@@ -259,6 +262,7 @@ resource service 'Microsoft.ApiManagement/service@2023-05-01-preview' = {
           minApiVersion: '2021-08-01'
         }
     restore: restore
+    developerPortalStatus: sku == 'Consumption' ? null : (enableDeveloperPortal ? 'Enabled' : 'Disabled')
```

```suggestion
    developerPortalStatus: sku != 'Consumption' ? (enableDeveloperPortal ? 'Enabled' : 'Disabled') : null
```
Have the null as the else case is usually a bit easier to read